ringpop_test.go

```diff
@@ -147,6 +153,96 @@ func (s *RingpopTestSuite) TestRingpopNotReady() {
 	s.False(s.ringpop.Ready())
 }
 
+// TestStateCreated tests that Ringpop is in a created state just after
+// instantiating.
+func (s *RingpopTestSuite) TestStateCreated() {
+	s.Equal(created, s.ringpop.getState())
+}
+
+// TestStateInitialized tests that Ringpop is in an initialized state after
+// a failed bootstrap attempt.
+func (s *RingpopTestSuite) TestStateInitialized() {
+
+	// Create channel and start listening so we can actually attempt to
+	// bootstrap
+	ch, _ := tchannel.NewChannel("test2", nil)
+	ch.ListenAndServe("127.0.0.1:0")
```

Comment:
If we `ListenAndServe` on TChannel, I feel there should be a corresponding `Close()`.